mypy/fastparse.py

```diff
@@ -762,6 +765,39 @@ def visit_ExtSlice(self, n: ast35.ExtSlice) -> Node:
     def visit_Index(self, n: ast35.Index) -> Node:
         return self.visit(n.value)
 
+    # PEP 492 nodes: 'async def', 'await', 'async for', 'async with'.
```

Comment:
This is a nit, but it'd be better if these were left in their original locations rather than separated out.  Currently, the `visit_*` functions in this file are in the same order as the nodes in the `Python.asdl`, which is a nice property to maintain.  Also, most of them are virtually identical to the non-async versions -- having them right next to each other will make it more likely they stay in sync.